### PR TITLE
release NetCDF file resources

### PR DESCRIPTION
**Pull request by electricsam**
NetCDF files are opened, but never closed.  These changes allow database classes to be used with PEP-343, the "with" statement.

### CODE REVIEW

**Comment by castelao** [comment]
@electricsam, thanks ! I'll review it tonight, maybe tomorrow. One step ahead, please add yourself in the `.zenodo` and the bottom of `AUTHORS.rst`.

**Comment by electricsam** [comment]
@castelao Just an FYI, there are still resource issues with NetCDF, but it is not with this code.  There is a memory leak in the underlying C library, that is pending resolution.

https://github.com/Unidata/netcdf4-python/issues/1021
https://github.com/Unidata/netcdf-c/issues/2626

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-version='0.8.14'
+version='0.8.15'
```
Comment (castelao) [inline VERSION]: @electricsam , please undo this version bump. The version now is dynamically updated, so we don't need to change it manually anymore. Thanks!

File: oceansdb/__init__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 __author__ = 'Guilherme Castelao'
 __email__ = 'guilherme@castelao.net'
-__version__ = '0.8.14'
+__version__ = '0.8.15'
```
Comment (castelao) [inline oceansdb/__init__.py]: Same here. Undoing this should resolve the conflict.

**Review by castelao — CHANGES_REQUESTED** [review]
Thanks!!

There is a conflict with the version definition. Look at my comments below. Also, please update with the changes I did upstream so it can run all the tests. Let me know if you would like help to do that.

**Comment by electricsam** [comment]
@castelao I reverted the version bump

**Review by castelao — APPROVED** [review]

**Comment by castelao** [comment]
@electricsam , thank you! I'll release a new version later today.

**Pull request merged**